create-event/src/components/Root.jsx

```diff
@@ -0,0 +1,38 @@
+/* @flow */
+
+import React from 'react'
+
+import Header from '../components/Header'
+import Routes from '../components/Routes'
+import DialogTextInput from '../components/DialogTextInput'
+import SnackbarContainer from '../containers/SnackbarContainer'
+
+const Root = props => {
+  const { user, paths, components, fetchUser } = props
+  if (!user.id) {
+    return (
+      <DialogTextInput
+        open
+        title="Enter Picatic API Key"
+        value={user.apiKey}
+        errorMessage={user.errorMessage}
+        placeholder="sk_live_210eb57e6b95e5143c492a219091c4e5"
```

Comment:
@tmirmota You're okay with using your actual private key here?

Reply:
It's not an admin key so I thought it would be easier for people to play around with. Easy to remove.